google-cloud-storage/src/test/java/com/google/cloud/storage/BucketTest.java

```diff
@@ -446,15 +444,17 @@ public void testCreateWithWrongGenerationOptions() throws Exception {
     replay(storage);
     initializeBucket();
     byte[] content = {0xD, 0xE, 0xA, 0xD};
-    thrown.expect(IllegalArgumentException.class);
-    thrown.expectMessage(
-        "Only one option of generationMatch, doesNotExist or generationNotMatch can be provided");
-    bucket.create(
-        "n",
-        content,
-        CONTENT_TYPE,
-        Bucket.BlobTargetOption.generationMatch(42L),
-        Bucket.BlobTargetOption.generationNotMatch(24L));
+    try {
+      bucket.create(
+          "n",
```

Comment:
Is this intentional? Couldn't we set some meaningful name like `test-bucket` or something?

Reply:
I have just removed deprected method and added try catch that's it